pep-0001.txt

```diff
@@ -148,10 +148,26 @@ The standard PEP workflow is:
 * You, the PEP author, fork the `PEP repository`_, and create a file named
   ``pep-9999.rst`` that contains your new PEP.  Use "9999" as your draft PEP
   number.
+
 * Push this to your GitHub fork and submit a pull request.
-* The PEP editors review your PR for structure, formatting, and other errors.
+
+* The PEP editors review your PR for structure, formatting, and other
```

Comment:
There's a template, PEP 12. I'd expect that there ought to be followed. Maybe mention it here?

Reply:
Yes!